xknx/io/request_response.py

```diff
@@ -16,10 +16,11 @@ class RequestResponse:
 
     # pylint: disable=too-many-instance-attributes
 
-    def __init__(self, xknx, udp_client, awaited_response_class, timeout_in_seconds=1):
+    def __init__(self, xknx, udp_client, awaited_response_class, timeout_in_seconds=1, net_bind=None):
         """Initialize RequstResponse class."""
         self.xknx = xknx
         self.udpclient = udp_client
+        self.net_bind = net_bind
```

Comment:
```suggestion
    def __init__(self, xknx, udp_client, awaited_response_class, timeout_in_seconds=1):
        """Initialize RequstResponse class."""
        self.xknx = xknx
        self.udpclient = udp_client
```
I'd remove this from the base class and just keep it local in the 3 classes where it is actually used.